test_files/comments/comments.tsickle.ts

```diff
@@ -14,13 +14,15 @@ export1: string;
   nodoc2: number;
   /// non js-doc comment
   nodoc3: number;
-  /** inline jsdoc comment without type annotation */
-  jsdoc1: number;
-  /**
-   * multi-line jsdoc comment without
-   * type annotation.
-   */
-  jsdoc2: number;
+/**
+ * inline jsdoc comment without type annotation
```

Comment:
I remember it taking some effort to figure out the right order of regexes/parsing to get whitespace right in comments, but I think it's probably not worth worrying about here.